data-model/src/main/scala/za/co/absa/enceladus/model/Dataset.scala

```diff
@@ -131,7 +131,7 @@ case class Dataset(name: String,
     val conformanceJsonList: ArrayNode = objectMapperBase.valueToTree(conformance.toArray)
     val propertiesJsonList: ObjectNode = objectMapperBase.valueToTree(propertiesAsMap)
 
-    val objectItemMapper = objectMapperRoot.`with`("item")
+    val objectItemMapper = objectMapperRoot.withObject("/item")
```

Comment:
This is a result of update to Jackson Databind 2.14 (both the method name and the json path).
(JsonNode.`with` being deprecated) 

https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.14#databind